docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+[[add-fleet-server-kubernetes]]
+= Deploy Fleet Server on Kubernetes
+
+[NOTE]
+====
+For ECK users, if your {stack} is orchestrated by ECK, we recommend to orchestrate the {fleet-server} through the operator. That simplifies the process as all needed resources will be configured by ECK.
++
+Refer to {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] for more information.
+====
+
+[IMPORTANT]
+====
+This guide assumes certain knowledge about deploying applications on Kubernetes.
```

Comment:
This guide assumes familiarity with deploying applications on Kubernetes. ?

"Certain knowledge" is interesting. I wonder what David thinks 👀

Reply:
Sorry I hadn't finished that sentence... I left only that as a placeholder to complete it at a later stage :)

Reply:
Text updated, let me know how it looks now, please @bmorelli25 :)

Reply:
much better!